Doc/library/pathlib.rst

```diff
@@ -1123,46 +1128,37 @@ call fails (for example because the path doesn't exist).
 
 .. method:: Path.is_symlink()
 
-   Return ``True`` if the path points to a symbolic link, ``False`` otherwise.
-
-   ``False`` is also returned if the path doesn't exist; other errors (such
-   as permission errors) are propagated.
+   Return ``True`` if the path points to an accessible symbolic link,
```

Comment:
Might be worth emphasising that this function does not follow symlinks? It's the odd one out, I believe.

Reply:
I added a bit about it returning `True` for broken links, which I think is a slightly more practical spin on the same idea. What do you think?